pywwt/traits.py

```diff
@@ -79,7 +79,7 @@ def __init__(self, *args, **kwargs):
             self.__doc__ = self.help
 
     def validate(self, obj, value):
-        if isinstance(value, str) or (isinstance(value, tuple) and len(value) == 3):
+        if isinstance(value, six.string_types) or (isinstance(value, tuple) and len(value) == 3):
```

Comment:
I think the import for ``six`` is missing? Just add ``import six`` and include ``six`` in the ``install_requires`` list in ``setup.py``.